foundations/html_css/block-and-inline.md

```diff
@@ -19,6 +19,18 @@ Inline elements, however, do not start on a new line. They appear in line with w
 
 Inline-block elements behave like inline elements, but with block-style padding and margin. Inline-block is a useful tool to know about, but in practice, you'll probably end up reaching for flexbox more often if you're trying to line up a bunch of boxes. Flexbox will be covered in-depth in the next lesson.
 
+Here's a table comparing and contrasting `block`, `inline-block`, and `inline`:
+
+|                                                                    |       `block`      |   `inline-block`   |          `inline`         |
+|:------------------------------------------------------------------:|:------------------:|:------------------:|:-------------------------:|
+|                            **behavior**                            |                    |                    |                           |
```

Comment:
The behaviour title looks a little odd in its own row. Should it be in the same row as the block and inline-block column titles?